### DISCUSSION TITLE
How to exclude adult movies?

### DISCUSSION THREAD

**Discussion by rougesteelproject** [cinemagoer/cinemagoer#392] [Q&A]
Is it `ia = Cinemagoer(adultSearch=0)`, or has the syntax changed?

**Comment by alberanid** [comment]
Well, pardon for the long delay.

By default they are not shown, and the `adultSearch` argument is no longer working and should be removed / transformed in a warning.

To do an adult search you can use the advanced search.

An example:
```
import imdb

ia = imdb.Cinemagoer('http')

for result in ia.search_movie_advanced('something really NSFW', adult=True):
    print(result)
```

hope this helps.

**Comment by alberanid** [comment]
the adult filter should work as expected in the s3 parser
